app/uk/gov/hmrc/apiplatform/modules/approvals/controllers/ApprovalsController.scala

```diff
@@ -44,7 +44,7 @@ object ApprovalsController {
   case class DeclinedRequest(gatekeeperUserName: String, reasons: String)
   implicit val readsDeclinedRequest = Json.reads[DeclinedRequest]
 
-  case class GrantedRequest(gatekeeperUserName: String, warnings: Option[String])
+  case class GrantedRequest(gatekeeperUserName: String, warnings: Option[String], escalatedBy: Option[String])
```

Comment:
If this is escalatedBy, does the userName contain the approver???  If not, should this read `escalatedTo` ?

Reply:
Good point - changed field name to EscalatedTo